index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Personal Portfolio</title>
+  <link rel="stylesheet" href="main.css" />
+ 
+</head>
+
+	<body>
+		<header class="Flexbox-header">
+			<div
+				class="flex-container header flex-row align-center justify-between"
+			>
+				<a href="#" class="logo color-primary">My Logo</a>
+				<svg
+					class="NavToggle"
+					width="24"
+					height="24"
+					viewBox="0 0 24 24"
+					fill="none"
+					xmlns="http://www.w3.org/2000/svg"
+				>
+				<path
+						fill-rule="evenodd"
+						clip-rule="evenodd"
+						d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
+						fill="#6070FF" >
+
+			</path>
+		</svg>
+				<nav class="navigation_head">
+					<ul class="navigation_list flex-row align-center">
+						<li class="navigation_list--item">
+							<a class="color-info" href="#">Portfolio</a>
+						</li>
+						<li class="navigation_list--item">
+							<a class="color-info" href="#">About</a>
+						</li>
+						<li class="navigation_list--item">
+							<a class="color-info" href="#">Contact</a>
+						</li>
+					</ul>
+				</nav>
+			</div>
+		</header>
+		<main class="main">
+			<div class="bg" id="flexbox-head-container"></div>
+			<section class="section section__white section__white--bottom">
+				<div class="headline flex-column justify-center">
+					<h1 class="headline__title color-dark">
+						I'm Asciencio.
+						<br />
+						Glad to see you!
+					</h1>
+					<p class="desc color-info">
+						I’m a software developer! I can help you build a product ,
+						feature or website Look through some of my work and experience!
+						If you like what you see and have a project you need coded,
+						don’t hesitate to contact me.
+					</p>
+					<a href="#">Let's Connect</a>
+					<div class="social__icons flex-row align-center">
+						<a href=""><img src="img/t_icon.png" alt="Twitter icon" /></a>
+						<a href=""><img src="img/Li_icon.png" alt="LinkedIn icon" /></a>
+						<a href=""><img src="img/m_icon.png" alt="Medium icon" /></a>
+						<a href=""><img src="img/_gth_icon.png" alt="Github icon" /></a>
+						<a href=""><img src="img/finger_c.png" alt="finger icon" /></a>
+					
+					</div>
```

Comment:
- [ ] Great job! On adding the icons inside an `<a>` tag, but I suggest that you remove the `<div>` & use a `<ul>` tag instead then store each icon inside a `<li>` tag.

Reply:
@NatiG25 thanks for the comment, let me change it.